.eslintrc

```diff
@@ -1,23 +1,25 @@
 {
   "extends" : [
-    "@visionappscz/eslint-config-visionapps",
-    "plugin:markdown/recommended",
-    "plugin:mdx/recommended"
+    "@visionappscz/eslint-config-visionapps"
   ],
   "env": {
     "browser": true,
     "jest": true
   },
-  "rules": {
-    "func-names": "off",
-    "import/prefer-default-export": "off",
-    "react/destructuring-assignment": "off",
-    "react/jsx-props-no-spreading": "off",
-    "react/jsx-one-expression-per-line": "off"
-  },
   "overrides": [
+    {
+      "files": ["**/*.md"],
```

Comment:
For unknown reason, those markdown plugins does not work when defined globally. If they are moved under `overrides`  section for MD or MDX, it works as expected. As it is not used by javascript linter by default, I think, it should be OK to have it here.

Reply:
Actually I think it can be simplified even further. I tested it and it seems that we use:
```
{
  "extends" : [
    "@visionappscz/eslint-config-visionapps"
  ],
  "env": {
    "browser": true,
    "jest": true
  },
  "overrides": [
    {
      "files": ["**/*.md", "**/*.mdx"],
      "extends": [
        "plugin:markdown/recommended",
        "plugin:mdx/recommended"
      ]
    }
  ],
  "parser": "@babel/eslint-parser",
  "rules": {
    "func-names": "off",
    "import/prefer-default-export": "off",
    "react/destructuring-assignment": "off",
    "react/jsx-props-no-spreading": "off",
    "react/jsx-one-expression-per-line": "off"
  }
}
```

It produces no errors and `*.md(x)` get validated. It seems that the override rules did nothing.